pkgs/tools/misc/lazydocker/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "lazydocker-${version}";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "jesseduffield";
+    repo = "lazydocker";
+    rev = "v${version}";
+    sha256 = "0f062xn58dbci22pg6y4ifcdfs8whzlv2jjprxxk2ygzixrrjwnc";
+  };
+
+  modSha256 = "02n0lg28icy11a2j2rrlmp70blby0kmjas5j48jwh9h9a0yplqic";
+
+  subPackages = [ "." ];
```

Comment:
Is this required? What if you remove it?

Reply:
If I remove this, more binaries will end up in `./result/bin/`:
```
$ ls -l /nix/store/0k6705dbhg3h7lzkbxdny1mlji07h4v6-lazydocker-0.5/bin/
total 23912
-r-xr-xr-x 1 root root 10627376 Jan  1  1970 lazydocker
-r-xr-xr-x 1 root root  1698816 Jan  1  1970 printrandom
-r-xr-xr-x 1 root root 10504528 Jan  1  1970 scripts
-r-xr-xr-x 1 root root  1648704 Jan  1  1970 testmain
```

With this, it will only be `lazydocker`:
```
ls -l /nix/store/yghmq3r0ypsq1mn5p80mka4knlk4llr3-lazydocker-0.5/bin/
total 10380
-r-xr-xr-x 1 root root 10627376 Jan  1  1970 lazydocker
```

See https://nixos.org/nixpkgs/manual/#ex-buildGoModule-2

Reply:
:+1: